amo2kinto/tests/test_importer.py

```diff
@@ -276,12 +276,22 @@ def test_sync_records_calls_the_scenario():
 
 class TestMain(unittest.TestCase):
     def setUp(self):
+        # Mock requests
         request_mock = mock.MagicMock()
         request_mock.json.return_value = RECORDS
         p = mock.patch('requests.get', return_value=request_mock)
         self.addCleanup(p.stop)
         p.start()
 
+        # Mock client
+        self.MockedClient = mock.MagicMock()
+        self.MockedClient.return_value.session.request.return_value = {
+            'capabilities': {}
+        }, {}
```

Comment:
If I remember well Boris talk about mock, we should instead define a class here that exposes the same signature as the original kinto-client instead of mocking internal stuff like that. This remark can be out of scope though and you can choose to ignore it ;)